packages/features/ee/organizations/pages/settings/general.tsx

```diff
@@ -87,6 +88,7 @@ const OrgGeneralView = () => {
       />
 
       <LockEventTypeSwitch currentOrg={currentOrg} isAdminOrOwner={!!isAdminOrOwner} />
+      <AdminNoSlotsNotificationSwitch currentOrg={currentOrg} isAdminOrOwner={!!isAdminOrOwner} />
```

Comment:
nit: I think we should rename the component to `NoSlotNotificationSwitch` as it takes `isAdminOrOwner`  that helps it to decide what should be shown.

Another way could be that we keep the name same(but no isAdminOrOwner prop. We can then instead conditionally render the component here based on isAdminOrOwner